### ISSUE TITLE
What happend to the api? is it down forever? will it come back?

### ISSUE THREAD

**Issue by mark909099** [apilayer/restcountries#253]
The url "https://restcountries.eu/rest/v2/all" doesn't give countries information.

It display error message: "The page you requested could not be found. Try refining your search, or use the navigation above to locate the post".

**Comment by fanetech** [comment]
how do?
i'have the same problem

**Comment by amatosg** [comment]
you can check this information

https://gitlab.com/amatos/rest-countries/-/blob/master/README.md

**Comment by Developer-Raman** [comment]
This now available in v3 version. 
Please check out [https://restcountries.com](https://restcountries.com/)

**Comment by mark909099** [comment]
> This now available in v3 version.
> Please check out [https://restcountries.com](https://restcountries.com/)

Thanks alot!

**Comment by AntonioChicano** [comment]
The https://restcountries.com/ API works very well minus the flags.
For some reason, the flags do not show them and I have checked my code but everything is fine.

because it can be?

**Comment by PaulRosset** [comment]
Thank you for forking the original repo!

Do also experience a slow response time for the URL `https://restcountries.com/` ?

**Comment by AntonioChicano** [comment]
Yes. Very slow

**Comment by amatosg** [comment]
you're right. Is taking way too long to respond. I'll take a look at that

**Comment by absolutecommunications** [comment]
Thank you, amatosg you rock!

**Comment by amatosg** [comment]
> Yes. Very slow

it looks like the CPU was at 100% all the time. Now is under 500ms

**Comment by ePascalC** [comment]
Message from apilayer support: We no longer support restcountries.eu; however, we do offer a free tier of a similar API on countrylayer.com. See https://countrylayer.com/documentation/

**Comment by maPer77** [comment]
Does anyone know since which day restcountries.eu is down?
I would like to know how many days my app has been broken...  :disappointed:

**Comment by ePascalC** [comment]
@maPer77 the first issue raised here was https://github.com/apilayer/restcountries/issues/250, so consider issues from that date.

**Comment by PaulRosset** [comment]
The Free tier fromm count

**Comment by chrismwalker-portfolio** [comment]
@amatosg What has happened to v2 of the API, for example [?](https://restcountries.com/v2/alpha/col). I updated my code to accommodate 'subregion' becoming 'region', and for 'flags' now being an array containing both SVG and PNG images, but today I notice v2 has reverted back to the old format response - just a single SVG flag, and 'subregion' is back again. Is this an intentional change, and will it now stay like this? Thanks for keeping the API alive!

**Comment by K2ouMais** [comment]
This API is not for free anymore. To be honest it is horrible now.

The documentation shows fields that arent there anymore.

Documentation:
![Countrylayer_Documentation](https://user-images.githubusercontent.com/52510455/135069000-1dbe5f30-1111-4a7d-8b22-5e6b22304ecf.png)

API Response:
![Countrylayer_Response](https://user-images.githubusercontent.com/52510455/135069062-ee9a2b21-f590-4a4b-afb6-a20835d92341.png)

This is a perfect example how it shouldnt be made.

**Comment by amatosg** [comment]
> @amatosg What has happened to v2 of the API, for example https://restcountries.com/v2/alpha/col. I updated my code to accommodate 'subregion' becoming 'region', and for 'flags' now being an array containing both SVG and PNG images, but today I notice v2 has reverted back to the old format response - just a single SVG flag, and 'subregion' is back again. Is this an intentional change, and will it now stay like this? Thanks for keeping the API alive!

@chrismwalker-portfolio yes, the changes was intentions. Sorry about that. The idea is to support the compatibility to this project so migration is *easier*

**Comment by amatosg** [comment]
> This API is dead for me at least.

this API is dead for everyone

**Comment by chrismwalker-portfolio** [comment]
> @chrismwalker-portfolio yes, the changes were intentional. Sorry about that. The idea is to support the compatibility to this project so migration is _easier_

No problem, it makes sense to keep it aligned to the original v2 API. Thanks again for all your work!

**Comment by PeterKassenaar** [comment]
@amatosg Thanks for the new fork/repo. Donated.

**Comment by amatosg** [comment]
I you have any issues, please report them [here](https://gitlab.com/amatos/rest-countries/-/issues)

**Comment by anoop4real** [comment]
OMG Will the API come back?...seem to be certificate issue

**Comment by amatosg** [comment]
> OMG Will the API come back?

No, but they offer a free tier where you can make 100 requests per month

**Comment by rwieruch** [comment]
@amatosg the website is down. Is there anything that we can to help?

**Comment by tricipitinus** [comment]
oh noes! we were actually hoping this would be the more stable option? Any word on what's the cause and how long it will take?
Why doesn't the UN offer this service anyway?

**Comment by chrismwalker-portfolio** [comment]
@amatosg I've raised it [here](https://gitlab.com/amatos/rest-countries/-/issues/79) as well. Thanks!

**Comment by weilinzung** [comment]
v3.1 is down now...

**Comment by ePascalC** [comment]
@weilinzung @tricipitinus @rwieruch, if you have an issue with that website, please create an issue on the dedicated issue log like @chrismwalker-portfolio did on GitLab https://gitlab.com/amatos/rest-countries/-/issues or eventually on the mirror on GitHub https://github.com/amatosg/restcountries/issues
Thanks.

**Comment by NizarBerjawi** [comment]
If you're looking for a substitute API check out: [https://placesapi.dev/](https://placesapi.dev/)

**Comment by gobinda-prasad-paudel** [comment]
Bro , after doing some search on internet I have found a result . The alternative or same of https://restcountries.eu/rest/v2/all is https://restcountries.com/v2/all

**Comment by nihalanisumit** [comment]
is this down again?

**Comment by sumithlogi** [comment]
Its unavailable again

**Comment by amatosg** [comment]
if you have issues with restcountries.com then please create a new issue here: https://gitlab.com/restcountries/restcountries/

**Comment by samathi-97** [comment]
> is this down again?

yes

**Comment by Ehtz** [comment]
``` https://restcountries.com/v3.1 ``` is down

**Comment by ExtAnimal** [comment]
restcountries.com is down:

![Screenshot 2024-06-06 at 17 10 45](https://github.com/apilayer/restcountries/assets/200516/2b02042e-2bdc-4527-b8dd-d7876b1ea30b)

**Comment by amatosg** [comment]
@ExtAnimal wrong repo
